types/jss/index.d.ts

```diff
@@ -104,13 +104,13 @@ export interface RuleOptions {
 	className: string;
 }
 export declare class SheetsRegistry {
-    constructor();
-    registry: ReadonlyArray<StyleSheet>;
-    readonly index: number;
-    add(sheet: StyleSheet): void;
-    reset(): void;
-    remove(sheet: StyleSheet): void;
-    toString(options?: ToCssOptions): string;
+	constructor();
+	registry: ReadonlyArray<StyleSheet>;
+	readonly index: number;
+	add(sheet: StyleSheet): void;
+	reset(): void;
+	remove(sheet: StyleSheet): void;
+	toString(options?: ToCssOptions): string;
```

Comment:
Any reason for the style change here?  I think the style on here was dictated by a linter.  In any case, it probably shouldn't change without reason (and 2 or 4 spaces is more common than 3).

Reply:
The rest of the file was tabs, while this part was spaces, so my editor re-formatted it automatically.